server/src/instrument/provider/display.ts

```diff
@@ -24,6 +24,17 @@ export const completions: Array<CompletionItem> = [
         kind: CompletionItemKind.Module,
         label: 'display'
     },
+    {
+        data: { domains: ['display'] },
+        documentation: {
+            kind: MarkupKind.Markdown,
+            value: '```lua\nactivebuffer\n```\n\
+\n\
+Get to set the buffer used for measurements that are displayed on the front panel.'
```

Comment:
`Get to set` -> `Get or set`

Does it accept/return a string? Does it accept/return a bufferVar? Type information is needed here. (See various `smu.*` namespaces for references)